spec/pQueue.js

```diff
@@ -0,0 +1,84 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import pQueue from '../src/pQueue'
+
+chai.use(chaiChange)
+
+describe('pQueue', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(pQueue).to.be.a('function')
+  })
+
+  context('enqueue()', () => {
+    it('adds an element with priority to the back of the queue.', () => {
+      const aQueue = new pQueue()
+
+      aQueue.enqueue('dude', 90)
+      aQueue.enqueue('cool', 20)
+      aQueue.enqueue('turd', 50)
+      expect(aQueue.enqueue('foo',70)).to.eql([{name: 'dude', priority: 90}, {name: 'cool',  priority: 20}, {name: 'turd', priority: 50}, {name: 'foo', priority: 70}])
+    })
+  })
+
+  // context('front()', () => {
+  //   it('returns the front element (highest priority) in the queue.', () => {
+  //     const aQueue = new pQueue()
+
+  //     aQueue.enqueue('baz', 300)
+  //     aQueue.enqueue('bar', 200)
+
+  //     expect(() => aQueue.front().to.eql([{name: 'baz', priority: 300}]))
+  //   })
+
+  //   it('returns null if the queue is empty.', () => {
+  //     const aQueue = new pQueue()
+
+  //     expect(() => aQueue.front().to.eql(null))
+  //   })
+  // })
+
+  // context('back()', () => {
+  //   it('returns the back element (lowest priority) in the queue.', () {
+  //     const aQueue = new pQueue()
+
+  //     aQueue.enqueue('bar', 200)
+  //     aQueue.enqueue('foo', 100)
+
+  //     expect(() => aQueue.back().to.eql([{name: 'foo', priority: 100}]))
+  //   })
+
+  //   it('returns null if the queue is empty.', () => {
+  //     const aQueue = new pQueue()
+
+  //     expect(() => pQueue.back().to.eql(null))
+  //   })
+  // })
+
+  context('isEmpty()', () => {
+    it('returns true if the queue is empty.', () => {
+      const aQueue = new pQueue()
+
+      expect(() => aQueue.isEmpty().to.eql(true))
```

Comment:
This is incorrect syntax.  Your test should be:
```js
expect( aQueue.isEmpty() ).to.be.true
```
The general format for a test is:
```js
expect( VALUE ).EXPECTATION
```
Where VALUE could be a return value from a function (like `isEmpty`), and EXPECTATION is the code that validates your expectation of that VALUE (`.to.be.true` or `.to.eql( true )`)